config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -21,6 +21,35 @@ presubmits:
         # Without account keys (-key-files), we should not be trying to activate
         # any service accounts.
         - -no-service-account
+  # Check that changes to backup scripts are valid.
+  - name: pull-k8sio-backup
+    decorate: true
+    skip_report: false
+    run_if_changed: 'infra/gcp/backup_tools/.*'
+    max_concurrency: 1
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191124-4beb966-1.17
+        command:
+        - infra/gcp/backup_tools/backup_test.sh
+      env:
```

Comment:
I think your indentation is off starting on this line.

Reply:
The other comments can be ignored, but I'm pretty sure this won't work as you expect.

Reply:
Ack

Reply:
For the record, I suppose it's troublesome that the presubmit checks passed regardless.

Reply:
It is a known issue, but I haven't addressed it out of fear that strictly parsing pod specs will fail if new fields that Prow doesn't know about are used. It may not be a legitimate concern though, it is kind of hard for me to tell if that would actually be a common problem.  It may be fine if 
- new podspec fields aren't used much in prowjobs
- we keep this dependency up to date and continue to keep prow.k8s.io up to date: https://github.com/kubernetes/test-infra/blob/44c22ec76a9c4280f9c977c7faf7fb49d860a652/repos.bzl#L1448-L1456